test/test_indexer_new.py

```diff
@@ -31,7 +31,7 @@ def test_bpe_indexer(self):
         from ml_models.bpe_model import indexer
         for q in questions:
             r = indexer.model.sentence_embedding(q)
-            self.assertTrue(r.shape[0]==300) # embedding dimension
+            self.assertEqual(r.shape[0], 300) # embedding dimension
```

Comment:
It's possible to add a dimension field to the indexer/model and use it at the test.
Models can have any dim